ome/omero-insight
# Add config to jar

Second phase
This PR includes config folder in the fat jar. This should make it easier for users to install the plugin
if you have previously installed omero.imagej in the plugins directory of ImageJ/Fiji, you first need to delete it.
To test with Fiji and ImageJ
 * If you build locally, you will under ``build/libs``, the ``omero_ij*.jar``. 
 * If you retrieve it from merge-ci, unzip ``imagej*.zip`` and retrieve the jar.  (When we tag, the ``zip`` will not be pushed, the ``jar`` will.
 * Place the omero_ij jar into the plugins directory.
 * Check that you can connect and view images etc.
 * Disconnect
 * ImageJ only, Check that there is no file ``logFileName_IS_UNDEFINED`` created in the folder containing ImageJ

cc @pwalczysko
